docs/examples.md

```diff
@@ -179,11 +181,43 @@ job('test') {
 
 ```
 
+### Running a job on a schedule
+
+```groovy
+job('example') {
+    triggers {
+       cron("H/15 * * * *")
+    }
+}
+```
+
+### Including a [Jenkins Custom Tool](https://wiki.jenkins-ci.org/display/JENKINS/Custom+Tools+Plugin)
+
+```groovy
+job('example') {
+    wrappers {
+       customTools(['jq'])
+    }
+}
+```
+
+### Including variables in a shell step
+
+```groovy
+var foo = 'bar';
+
+job('example') {
+    steps {
+       shell("my_command --my-arg ${foo}")
```

Comment:
you might want to point out how to use jenkins variables too, b/c those just need to be escaped

i.e.,
```
shell("my_command --my-arg \${jenkins_variable_not_defined_in_script}")
```